Add mention to dino.icu domains in scrapbook (#374)
hackclub/scrapbook: README.md

To put your profile on your own domain, run `/scrappy setdomain <domain>` in Slack, giving your website’s hostname (e.g. [`scrapbook.maggieliu.dev`](https://scrapbook.maggieliu.dev)). Then, add a `CNAME` record on your DNS provider, pointed to `cname.vercel-dns.com`. If you’re curious how this works, it’s [open source right here](http://github.com/hackclub/summer-domains).

> Hack Clubbers can also use a `dino.icu` subdomain! You can use this domain for your scrapbook or for another project. To get one, go to our [DNS repository](https://github.com/hackclub/dns), where we manage our DNS configuration through OctoDNS and there follow the instructions to obtain it, after that you can run `/scrappy setdomain <domain>` from instructions above.

Reviewer: ```suggestion
Hack Clubbers can also use a `dino.icu` subdomain! You can use this domain for your Scrapbook or for another project. To get one, go to our [DNS repository](https://github.com/hackclub/dns), where we manage our DNS configuration through OctoDNS and there follow the instructions to obtain it, after that you can run `/scrappy setdomain <domain>` from instructions above.
```

> Hack Clubbers can also use a `dino.icu` subdomain! You can use this domain for your Scrapbook or for another project. To get one, go to our [DNS repository](https://github.com/hackclub/dns), where we manage our DNS configuration through OctoDNS and there follow the instructions to obtain it, after that you can run `/scrappy setdomain <domain>` from instructions above.

Reviewer: ```suggestion
Hack Clubbers can also use a `dino.icu` subdomain by submitting a PR to our [DNS repository](https://github.com/hackclub/dns).
```

I'd like to shorten it a bit

Reviewer: (ping me on slack after this)

> Hack Clubbers can also apply for a very cool `dino.icu` domain! You can use this domain for your scrapbook or for another project. To get one, go to our [DNS repository](https://github.com/hackclub/dns), where we manage our DNS configuration through OctoDNS and there follow the instructions to obtain it, after that you can run `/scrappy setdomain <domain>` from instructions above!

Reviewer: ```suggestion
Hack Clubbers can also use a `dino.icu` subdomain! You can use this domain for your scrapbook or for another project. To get one, go to our [DNS repository](https://github.com/hackclub/dns), where we manage our DNS configuration through OctoDNS and there follow the instructions to obtain it, after that you can run `/scrappy setdomain <domain>` from instructions above.
```
